data/geany-3.0.css

```diff
@@ -1,10 +0,0 @@
-@import "geany.css";
-
-/* make close button on the editor's tabs smaller */
-#geany-close-tab-button {
-	-GtkWidget-focus-padding: 0;
-	-GtkWidget-focus-line-width: 0;
-	-GtkButton-default-border: 0;
-	-GtkButton-default-outside-border: 0;
-	-GtkButton-inner-border: 0;
-}
```

Comment:
If I wanted to be somewhat unfair, give me a snippet that styles only the file open fialog in all GTK apps :)